chroma_core/migrations/0026_runstratagemjob_search_expression.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-08-31 17:05
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('chroma_core', '0025_createsnapshotjob_destroysnapshotjob'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='runstratagemjob',
+            name='search_expression',
+            field=models.TextField(default=''),
+            preserve_default=False,
+        ),
+    ]
```

Comment:
This appears to be missing an added field.

Reply:
done